### ISSUE TITLE
Does it really need netstandard1.4? 

### ISSUE THREAD

**Issue by TsengSR** [damienbod/ElasticsearchCRUD#165]
Do you really need `netstandard1.4` for [one single API call](https://github.com/damienbod/ElasticsearchCRUD/blob/release2.4.1.1/src/ElasticsearchCrud/ElasticSearchContext.cs#L80)? 

I just tried to compile it and that's the only error. That's a pity to bump the framework by two versions, for one single API. 

Targeting `netstandard1.0` would be pretty nice as it allows it to run on more platforms or at least `netstandard1.2` which is 4.5.2. 

There are valid reasons still targeting 4.5.2, as some are limited to use it and 4.6/4.6.1 is not spread wide enough. 

One reason for example is [Orleans ](https://github.com/dotnet/orleans) and with it Orleankka which is based on it. They still require 4.5.2 and don't run on 4.6 and there won't be an upgrade to it until the penetration of 4.6.x is high enough. 

Orleans is commonly used in enterprise businesses and having such an library target 4.6.1 because of a single API seems a bit over the top.

**Comment by damienbod** [comment]
No, I would like to update netstandard1.0 seems like the right idea now. Also all need to clean up the tests again, will be probably change back to Nunit, because I have problems running the xunit in parallel. 

Will also update to Elasticsearch 5.xx.

Greetings Damien
